src/WebEid.AspNetCore.Example/Controllers/Api/AuthController.cs

```diff
@@ -29,32 +29,54 @@
     using System.Threading.Tasks;
     using Security.Challenge;
     using WebEid.AspNetCore.Example.Dto;
+    using System;
+    using Microsoft.Extensions.Logging;
 
     [Route("[controller]")]
     [ApiController]
     public class AuthController : BaseController
     {
         private readonly IAuthTokenValidator authTokenValidator;
         private readonly IChallengeNonceStore challengeNonceStore;
+        private readonly ILogger logger;
 
-        public AuthController(IAuthTokenValidator authTokenValidator, IChallengeNonceStore challengeNonceStore)
+        public AuthController(IAuthTokenValidator authTokenValidator, IChallengeNonceStore challengeNonceStore, ILogger logger)
         {
             this.authTokenValidator = authTokenValidator;
             this.challengeNonceStore = challengeNonceStore;
+            this.logger = logger;
         }
 
         [HttpPost]
         [Route("login")]
         public async Task Login([FromBody] AuthenticateRequestDto authToken)
         {
-            var certificate = await this.authTokenValidator.Validate(authToken.AuthToken, this.challengeNonceStore.GetAndRemove().Base64EncodedNonce);
-            var claims = new List<Claim>
+            var certificate = await authTokenValidator.Validate(authToken.AuthToken, challengeNonceStore.GetAndRemove().Base64EncodedNonce);
+
+            Dictionary<string, Func<string>> claimDataGetters = new()
```

Comment:
Since Dictionary is unordered and Claims in ClaimsIdentity are ordered, we should try to maintain that order.